packages/client/src/containers/apiTokenPage.tsx

```diff
@@ -14,173 +27,178 @@ const breadcrumbs: BreadcrumbItemSetup[] = [
     title: 'API token',
     tips: 'A valid token is mandatory to allow 3rd-party using PrimeHub APIs.',
     tipsLink: 'https://docs.primehub.io/docs/tasks/api-token'
-  }
+  },
 ];
 
-type Props = {
-  client: ApolloClient<any>;
+interface Props {
+  downloadConfig?: (config) => void; // This is only used in unit test
```

Comment:
If you want to mock download behavior, you can use `jest.mock` method to mock `FileSaver.saveAs` instead of given an unnecessary function as a prop.

You can reference [this way](https://stackoverflow.com/a/44692287)

Reply:
Fixed